docs/en/usage/apollo-open-api-platform.md

```diff
@@ -605,6 +605,47 @@ This interface is the interface used to get whether the current namespace is loc
 }
 ```
 
+##### 3.2.17 Create App And grant administrative privileges
+
+App can be created through this interface, 
+
+>  note: 需要在创建第三方应用时，**勾选允许创建app，否则会触发异常，HTTP状态码401**
```

Comment:
```suggestion
```

Should this line be deleted?

Reply:
I think we need to prompt user, some users maybe use 'old token' to call this openapi method and get the failed result.

Reply:
I concur. My suggestion is to remove the line in Chinese since it's already duplicated in English below.

Reply:
My fault. Remove it already.